src/main/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/repository/ReferenceDataRepository.kt

```diff
@@ -19,6 +19,13 @@ interface ReferenceDataRepository : JpaRepository<ReferenceData, Long> {
   fun findByDomainAndCodeIn(domain: ReferenceDataType, code: Set<String>): Collection<ReferenceData>
 }
 
+fun ReferenceDataRepository.getReferenceData(type: ReferenceDataType, code: String) =
```

Comment:
Possibly rename this `getActiveReferenceData` or even `getActiveReferenceDataOrThrow` to make it clear that it will only return active ones